src/rtp_sender.rs

```diff
@@ -27,35 +33,82 @@ impl PipelineOutput for RtpSender {
     type Context = RtpContext;
 
     fn new(options: Options, codec: Codec) -> Result<(Self, RtpContext), CustomError> {
-        let port = options.port;
-        let ip = options.ip.clone();
-
-        let mut output_ctx = format::output_as(
-            &PathBuf::from(format!(
-                "rtp://{}:{}?rtcpport={}",
-                options.ip, options.port, options.port
-            )),
-            "rtp",
-        )
-        .map_err(|e| CustomError(e.into()))?;
-
-        let mut stream = output_ctx
-            .add_stream(codec)
-            .map_err(|e| CustomError(e.into()))?;
-        unsafe {
-            (*(*stream.as_mut_ptr()).codecpar).codec_id = codec::Id::H264.into();
+        if codec.id() != codec::Id::H264 {
+            unimplemented!("Only H264 is supported");
         }
 
-        output_ctx
-            .write_header()
+        let mut rng = rand::thread_rng();
+        let ssrc = rng.gen::<u32>();
+        let next_sequence_number = rng.gen::<u16>();
```

Comment:
Do we have to start with a random sequence number? AFAIK RFC recommends this, but I don't see any benefits from starting at a random number

Reply:
I don't see any harm from doing that either. It's very straight-forward to do and doesn't make the implementation slower and the rfc recommends it. Why would we not do it?